src/setupEmbedRelay.ts

```diff
@@ -99,6 +102,11 @@ export function setupEmbedRelay({
       variables?: Record<string, string>;
       headers?: Record<string, string>;
     } = event.data;
+    // When embed connects, send a handshake message
+    if (data.name === EXPLORER_LISTENING_FOR_HANDSHAKE) {
+      sendHandshakeToEmbed(!schema ? 'graphRef' : 'schema');
```

Comment:
hmm, so we would want the actual graphRef to be sent here, not just the string `"graphRef"`.